src/parser/pegen.ts

```diff
@@ -2149,8 +2270,12 @@ export function keyword_or_starred(p: Parser, element: any, is_keyword: boolean)
 //     return n;
 // }
 
-export function seq_extract_starred_exprs(p: Parser, kwargs: KeywordOrStarred[]) {
-    return kwargs.filter((k) => k.is_keyword);
+function isKeyword(kw: expr | keyword): kw is keyword {
+    return kw instanceof keyword;
+}
```

Comment:
our first typeguard

Reply:
`kw instanceof keyword` is probably a typeguard in itself